docs/introduction.rst

```diff
@@ -1,73 +1,57 @@
 
-See :ref:`data-management`.
+Selecting more than one display panel will display with a solid blue rectangle around the primary selection and with dashed lines around the secondary selections. If a computation or function can only use one input, the primary selection will be used. For computations using multiple inputs, secondary selections may be used as well.
 
-Workspaces
-----------
-The main window is organized around a main area called a workspace. A workspace has a layout which includes one or more display panels in which display items can be visualized.
+Selecting multiple data items will allow you to move them at the same time. However, multiple data items cannot be dragged into the same display panel.
 
-You can create additional workspace for different uses and easily switch between them.
+Selecting multiple graphics in the same display panel will allow you to move them at the same time. The inspector panel will display options for all of the selected graphics.
 
-In addition, the main window has a variety of utility panels attached or associated with the main window. Some examples of utility panels are the data browser panel, the inspector panel, and the info and histogram panels.
+Graphics and Masks
+==================
+Nion Swift provides plenty of useful graphics to narrow down your view of the data. Use graphics to highlight portions of the data, select areas for processing, or to focus on specific sections of data.
 
-See :ref:`user-interface`.
+Masking allows you to block out the rest of an image and see only what you want. Choose from a variety of mask shapes to precisely define your area of interest.
 
-Display Panels
---------------
-Workspaces are split into one or more display panels. Each display panel can show one display item.
+See :ref:`graphics` for more information.
 
-A typical display item would show a single data item, but you can create multiple display items for each data item to visualize the data differently at the same time.
+Processing and Analysis
+=======================
+Processing and analysis greatly expand the functionality of Nion Swift. The software provides many options for filtering, transforming, and more. Computations allow for specific and customizable processes, and there is even support for python scripting to add your own computations.
 
-In addition, some display items may display multiple data items at once: a line plot may have multiple layers, each displaying a different data item or even different reductions of the same data item.
+See :ref:`processing` for more information.
 
-See :ref:`display-panels`.
+File Management
+===============
+Make use of Nion Swift without live acquisition by importing your own data files into the program, take your files with you by exporting your data, save your project files and use them on any other installation of Nion Swift, make copies of your data to create a backup, and contact us about recovery options.
```

Comment:
The sentence reads like bullet points.

"program" -> "application".

One idea to avoid using "program/application" is to say something like "importing your own data files into a project".